launchflow/node.py

```diff
@@ -133,14 +155,96 @@ class NodeType(Enum):
 T = TypeVar("T", bound=Outputs)
 
 
+# This function was pulled from this Stackoverflow answer:
+# https://stackoverflow.com/questions/57706180/generict-base-class-how-to-get-type-of-t-from-within-instance/71720366#answer-76250220
+def get_generic_map(
```

Comment:
this is gross lol

Reply:
yeah...we should definitely revisit this and see if we can simplify it, or just remove the need for it altogether